dvc/command/run.py

```diff
@@ -162,6 +164,19 @@ def add_parser(subparsers, parent_parser):
         help="Declare output plot file (do not put into DVC cache).",
         metavar="<path>",
     )
+    run_parser.add_argument(
+        "--dvclive",
+        action="append",
+        default=[],
+        help=argparse.SUPPRESS,
+        metavar="<path>",
+    )
+    run_parser.add_argument(
+        "--dvclive-no-summary",
+        action="store_true",
+        default=False,
+        help=argparse.SUPPRESS,
+    )
```

Comment:
Maybe `--live`? Also, you use `append`, but only one live output is allowed right now, right?

Reply:
I would argue that in case of user facing interface its good to go with full name, so there is no need in remembering that "`dvclive` type of output is named as `live` in dvc itself". Though this is more elegant. I will change it, we can always redo it later.